client/components/PrimaryCollectionExplanation/PrimaryCollectionExplanation.tsx

```diff
@@ -15,13 +15,9 @@ const copy = (
 const PrimaryCollectionExplanation = (props: Props) => {
 	const { children = 'Primary Collection' } = props;
 	return (
-		<>
-			<React.Fragment>
-				<Tooltip content={copy}>
-					<span className={Classes.TOOLTIP_INDICATOR}>{children}</span>
-				</Tooltip>
-			</React.Fragment>
-		</>
+		<Tooltip content={copy}>
+			<span className={Classes.TOOLTIP_INDICATOR}>{children}</span>
+		</Tooltip>
```

Comment:
eslint seems to have changed this. (i ran `eslint --fix` to fix all the prettier changes, this may have been caught in that)

I think this shouldn't do anything, unless you were somehow relying on some evil react thingy